server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/cucumber/CassandraStepdefs.java

```diff
@@ -53,8 +59,16 @@ public void init() throws Exception {
 
     @After
     public void tearDown() {
-        mainStepdefs.tearDown();
-        embeddedElasticSearch.after();
-        temporaryFolder.delete();
+        tearDown(() -> mainStepdefs.tearDown());
+        tearDown(() -> embeddedElasticSearch.after());
+        tearDown(() -> temporaryFolder.delete());
```

Comment:
What is the advantage on simply call the methods? Only independent exception ignoring?